### DISCUSSION TITLE
Rotate around a point that isn't the center of the image keeping the dimensions

### DISCUSSION THREAD

**Discussion by christiancascone** [dlemstra/Magick.NET#1390] [Help]
Hi,

I would like to rotate an image around a point that is not the center of the image. The resulting image should have the same dimensions as the original image, but only the content should be rotated.

Thanks,
Christian

**Comment by dlemstra** [comment]
I am not sure what you want to do. Could you ask your question here also: https://github.com/ImageMagick/ImageMagick/discussions? The answer they will give you will probably be something on the command line. If you need help translating that to C# then please come back here for help. And when you ask the question there it will help if you provide an input image and an output example.
